src/seml/commands/migration.py

```diff
@@ -0,0 +1,166 @@
+import logging
+from typing import TYPE_CHECKING, Protocol
+from seml.database import get_collection
+from seml.settings import SETTINGS
+from seml.utils import s_if, smaller_than_version_filter
+from seml.utils.slurm import get_cluster_name
+
+
+if TYPE_CHECKING:
+    from pymongo.collection import Collection
+
+States = SETTINGS.STATES
+
+
+def migrate_collection(db_collection_name: str, skip: bool, backup: bool):
+    """
+    Migrate the given collection to be compatible with the current SEML version.
+
+    Parameters
+    ----------
+    db_collection_name : str
+        The name of the collection to migrate.
+    skip : bool
+        Whether to skip the migration.
+    backup : bool
+        Whether to create a backup of the collection before migrating.
+    """
+    from seml.console import prompt
+
+    collection = get_collection(db_collection_name)
+
+    if skip:
```

Comment:
This is weird, the `skip` should be whenever the method is called, right?

Reply:
what do you mean?